pkg/mkverification-raw-efi/verify

```diff
@@ -411,6 +411,8 @@ else
    echo "No watchdogs available" > "$REPORT/watchdogs.log"
 fi
 
+/vmcap > "$REPORT/vmcap.log"
```

Comment:
I think we should check for the platform before running an x86-specific tool here. Something like:
```bash
ARCH=$(uname -m)
if [ "$ARCH" = "x86_64" ]; then
    /vmcap > "$REPORT/vmcap.log"
fi
```